renovate.json

```diff
@@ -8,6 +8,7 @@
     ]
   },
   "statusCheckVerify": true,
+  "commitMessagePrefix": "📦",
```

Comment:
Should this be `"📦 "`?

Reply:
Nope, I checked to make sure Renovate adds a space after the prefix.